lib/services/serviceBus/package.json

```diff
@@ -27,7 +27,7 @@
   "dependencies": {
     "azure-common": "^0.9.19",
     "mpns": "2.1.3",
-    "underscore": "^1.8.3",
```

Comment:
Will you also increase the version of serviceBus?

Reply:
no need, because there is an update which updated it.